metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -290,29 +278,43 @@ def loop_views(
                     self.report.report_warning(
                         f"{schema}.{view}", f"Ingestion error: {e}"
                     )
-                if sql_config.include_view_lineage:  # type: ignore
+                if self.config.include_view_lineage:
                     try:
                         dataset_urn = make_dataset_urn_with_platform_instance(
                             self.platform,
                             dataset_name,
                             self.config.platform_instance,
                             self.config.env,
                         )
+
                         dataset_snapshot = DatasetSnapshot(
                             urn=dataset_urn, aspects=[StatusClass(removed=False)]
                         )
+
                         lineage_info = self._get_upstream_lineage_info(
-                            dataset_urn, view
+                            dataset_urn,
+                            inspector,
+                            view,
+                            schema,
                         )
+
                         if lineage_info is not None:
-                            # upstream_column_props = []
-                            yield MetadataChangeProposalWrapper(
+                            upstream_lineage = lineage_info
+
+                            lineage_mcpw = MetadataChangeProposalWrapper(
                                 entityType="dataset",
                                 changeType=ChangeTypeClass.UPSERT,
                                 entityUrn=dataset_snapshot.urn,
                                 aspectName="upstreamLineage",
-                                aspect=lineage_info,
-                            ).as_workunit()
```

Comment:
revert this change

we no longer need to call report_workunity and `.as_workunit()` is cleaner

Reply:
done